libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -199,6 +199,11 @@ const AP_Param::GroupInfo AP_Mount::var_info[] PROGMEM = {
     // @User: Standard
     AP_GROUPINFO("_TYPE", 19, AP_Mount, state[0]._type, 0),
 
+    AP_GROUPINFO("_OFF_JNT", 20, AP_Mount, state[0]._gimbalParams.joint_angles_offsets, 0),
```

Comment:
The offsets will probably need to be moved to the gimbal itself. SO the units are swap-able without requiring re-calibration ( the gimbal keeps it's calibration parameters)